.github/workflows/wfc_lifecycle.yml

```diff
@@ -62,18 +62,20 @@ jobs:
         env:
           COMMENT: ${{ inputs.openComment }}
   autoclose:
-    if: ${{ github.event.action == 'labeled' && contains(inputs.autocloseLabels, github.event.label.name) && github.event.issue.state != 'closed' }}
+    if: ${{ github.event.action == 'labeled' && contains(inputs.autocloseLabels, github.event.label.name) }}
     env:
       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     runs-on: ubuntu-latest
     steps:
       - name: comment on issue
+        if: ${{ github.event.issue.state != 'closed' }}
```

Comment:
It occurs to me that for `if` statements there's no need for `${{ }}` delimiters (for next time)